package.json

```
@@ -78,7 +78,7 @@
 		"jetpack-js-tools": "workspace:*"
 	},
 	"engines": {
-		"node": "^14.17.6 || ^16.7.0",
+		"node": "^14.17.6 || ^16.13.2",
```

```suggestion
		"node": "^14.18.3 || ^16.13.2",
```

Should we bump this to the 14.x release that shipped along side 16.13.2?